api/envoy/extensions/compression/common/compressor/v3/compressor.proto

```diff
@@ -1,22 +0,0 @@
-syntax = "proto3";
-
-package envoy.extensions.compression.common.compressor.v3;
-
-import "google/protobuf/any.proto";
-
-import "udpa/annotations/migrate.proto";
-import "udpa/annotations/status.proto";
-import "udpa/annotations/versioning.proto";
-
-option java_package = "io.envoyproxy.envoy.extensions.compression.common.compressor.v3";
-option java_outer_classname = "CompressorProto";
-option java_multiple_files = true;
-option (udpa.annotations.file_status).package_version_status = ACTIVE;
-
-// [#protodoc-title: Compressor Library]
-
-message CompressorLibrary {
```

Comment:
I guess the point of this message was to potentially have fields that were common to all compressor extensions. However, if we believe there won't be a case for this, I am happy to delete.